deploy/helm/azure-industrial-iot/values.yaml

```diff
@@ -5,12 +5,12 @@ image:
   # image.registry is URL of Docker registry from where images will be pulled.
   registry: mcr.microsoft.com
   # image.tag defines which version of Docker images to pull.
-  tag: 2.5.2
+  tag: 2.6.104
```

Comment:
>104 [](start = 11, length = 3)

can't we use :latest respective :preview here?

Reply:
We shouldn't. Helm best practices explicitly state not to use 'floating' image tags and to always prefer fixed ones.

We should keep in mind that in Kubernetes environment different components may fail and be restarted independently. Each restart can result in a new image being pulled and used. So having a floating tag may then result in a situation where you have components of several different versions in the same environment. Also, having a floating tag would mean not having reproducible deployments, which should also be avoided.

Reply:
BTW, this does not mean that users cannot change the tag. Quite the opposite, it is very easy to override this value. But then users would have to explicitly do that at their own risk.